fixing extended trial dates (#9316)
fishtown-analytics/docs.getdbt.com: website/snippets/_sao-deprecated.md

[dbt State](/docs/deploy/dbt-state-about) works with all engines and environments: <Constant name="core" />, <Constant name="dbt_platform" />, and <Constant name="fusion" />

> If you're using state-aware orchestration prior to June 1, 2026, you can continue using it. As an existing customer, your dbt State trial will be extended until the billing period begins on September 1, 2026. If you don't see the extension, contact your account team. To get started, refer to [Migrate from state-aware orchestration](/docs/deploy/dbt-state-migration).

Reviewer: ```suggestion
If you're using state-aware orchestration prior to June 1, 2026, you can continue using it. Your dbt State trial will be extended until the billing period begins on September 1, 2026. If your trial wasn't extended, contact your account team. To get started, refer to [Migrate from state-aware orchestration](/docs/deploy/dbt-state-migration).
```
